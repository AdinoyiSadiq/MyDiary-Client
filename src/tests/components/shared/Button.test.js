import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Button from '../../../components/shared/Button';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<Button />);
});

describe('Button UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});