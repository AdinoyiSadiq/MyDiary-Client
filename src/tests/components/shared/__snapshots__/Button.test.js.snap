// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button UI render features view should render as expected 1`] = `<button />`;
